README.md

```diff
@@ -42,7 +42,7 @@ If you don't have Python installed, follow the instructions in the
 Clone the repository, change to the directory containing the repository.
 
 ```bash
-$ pip install -r requirements.txt
+$ pip install -r requirements.txt --user
```

Comment:
That's written in line below. I guess that depends very much on the setup. I guess most people these day use a virtualenv anyhow and this can brak virtualenv.